features/step_definitions/oauth_client.rb

```diff
@@ -2,3 +2,13 @@
   cb.cluster_resource_to_recreate = o_auth_client(name)
   step 'hidden recreate cluster resource after scenario'
 end
+
+Given(/^convert token (.*) to oauthaccesstoken successfully$/) do |token|
```

Comment:
@yaoli-redhat can you pls given an example of how a user would use this step?  Probably need to be more explicit in the step name that it's implicitly saving the converted result into the clipboard

Reply:
also probably missing a pronoun there, like `I` or `admin` before `convert token`

Reply:
@pruan-rht this is to support OCP-33656 auto case. You can check line 31 in https://github.com/openshift/cucushift/pull/8597 and the function is to conver user token to oauthaccesstoken.

Reply:
@pruan-rht update the step name, please help to review. Thanks. Pass log: https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/ocp-common/job/Runner/80291/console